CHANGELOG.rst

```diff
@@ -30,6 +30,7 @@ Completions
   - Angular's ``ng`` (:issue:`8111`)
   - ``zef`` (:issue:`8114`)
   - ``rakudo`` (:issue:`8113`)
+  - ``az``
```

Comment:
Since you now know this is PR 8141, you can add the issue number to it as well.

Reply:
Done